examples/camera-app/linux/src/clusters/webrtc-provider/webrtc-provider-manager.cpp

```diff
@@ -433,7 +444,15 @@ CHIP_ERROR WebRTCProviderManager::HandleEndSession(uint16_t sessionId, WebRTCEnd
         // TODO: Lookup the sessionID to get the Video/Audio StreamID
         ReleaseAudioVideoStreams();
 
-        mWebrtcTransportMap.erase(sessionId);
+        mMediaController->UnregisterTransport(it->second.get());
+        mWebrtcTransportMap.erase(it);
```

Comment:
Additionally, the CurrentSessions attribute also needs updating to clear the session from the attribute.